Error: Unable to import `../package.json`

I'm not sure what is causing this, my webpack build chain functions normally, but when you then load the app in the browser you get the error. Perhaps it's an issue with using relative paths and then in dev mode it producing a relative path that the browser is trying to open.

By updating https://github.com/elastic/search-ui/blob/3cfe66660a0b976443122160a4ec28a7b8a1ca5e/packages/search-ui-app-search-connector/src/AppSearchAPIConnector.js#L2 to:
```
const version = '1.5.0';
```

I was able to get around the issue as a very temporary patch

Edit:
Also tried to update it to: `import { version } from "@elastic/search-ui-app-search-connector/package.json";` which still produced the same issue:
```
Uncaught Error: Cannot find module '@elastic/search-ui-app-search-connector/package.json'
    at webpackMissingModule (package.json:1)
    at eval (package.json:1)
    at Object../node_modules/@elastic/search-ui-app-search-connector/package.json (bootstrap:83)
    at __webpack_require__ (bootstrap:19)
    at eval (AppSearchAPIConnector.js:16)
    at Module../node_modules/@elastic/search-ui-app-search-connector/es/AppSearchAPIConnector.js (bootstrap:83)
    at __webpack_require__ (bootstrap:19)
    at eval (index.js:2)
    at Module../node_modules/@elastic/search-ui-app-search-connector/es/index.js (bootstrap:83)
    at __webpack_require__ (bootstrap:19)
```

Edit 2:
Looks like the fool is me, babel was trying to read it and, then not being able too, so it would just exclude it from the build.
Updating my babel config to ignore `*/**/package.json` fixed the issue for me

--- a/packages/search-ui-app-search-connector/src/AppSearchAPIConnector.js
+++ b/packages/search-ui-app-search-connector/src/AppSearchAPIConnector.js
@@ -1,5 +1,4 @@
 import * as ElasticAppSearch from "@elastic/app-search-javascript";
-import { version } from "../package.json";
 
 import { adaptResponse } from "./responseAdapter";
 import { adaptRequest } from "./requestAdapters";
@@ -67,10 +66,6 @@ class AppSearchAPIConnector {
       ...(hostIdentifier && { hostIdentifier: hostIdentifier }),
       apiKey: searchKey,
       engineName: engineName,
-      additionalHeaders: {
-        "x-swiftype-integration": "search-ui",
-        "x-swiftype-integration-version": version
-      },
       ...rest
     });
     this.beforeSearchCall = beforeSearchCall;

--- a/packages/search-ui-site-search-connector/src/SiteSearchAPIConnector.js
+++ b/packages/search-ui-site-search-connector/src/SiteSearchAPIConnector.js
@@ -1,14 +1,8 @@
 import adaptRequest from "./requestAdapter";
 import adaptResponse from "./responseAdapter";
 import request from "./request";
-import { version } from "../package.json";
 
 function _get(engineKey, path, params) {
-  const headers = new Headers({
-    "x-swiftype-integration": "search-ui",
-    "x-swiftype-integration-version": version
-  });
-
   const query = Object.entries({ engine_key: engineKey, ...params })
     .map(([paramName, paramValue]) => {
       return `${paramName}=${encodeURIComponent(paramValue)}`;
@@ -19,8 +13,7 @@ function _get(engineKey, path, params) {
     `https://search-api.swiftype.com/api/v1/public/${path}?${query}`,
     {
       method: "GET",
-      credentials: "include",
-      headers
+      credentials: "include"
     }
   );
 }

--- a/packages/search-ui-site-search-connector/src/request.js
+++ b/packages/search-ui-site-search-connector/src/request.js
@@ -1,10 +1,6 @@
-import { version } from "../package.json";
-
 export default async function request(engineKey, method, path, params) {
   const headers = new Headers({
-    "Content-Type": "application/json",
-    "x-swiftype-integration": "search-ui",
-    "x-swiftype-integration-version": version
+    "Content-Type": "application/json"
   });
 
   const response = await fetch(

--- a/packages/search-ui/README.md
+++ b/packages/search-ui/README.md
@@ -54,16 +54,3 @@ of the Advanced README.
 | `subscribeToStateChanges` | function |                                      | Function to execute when state changes. ex.<br/><br/>`(state) => {}` |
 | `getActions`              |          | [Actions](../../ADVANCED.md#actions) | All available actions.                                               |
 | `getState`                |          | [State](../../ADVANCED.md#state)     | Current state.                                                       |
-
-### Does Search UI use telemetry?
-
-If you are using the App Search or Site Search connector, we pass along 2 headers on API requests
-that identify them as Search UI requests. This ONLY happens if you are using our pre-built
-connectors.
-
-Ex.
-
-```
-x-swiftype-integration: search-ui
-x-swiftype-integration-version: 0.6.0
-```

